codegen-client-test/model/error-correction-nullability-test.smithy

```diff
@@ -27,6 +27,10 @@ structure Error {
 
     @required
     message: String
+
+    code: String = "400"
```

Comment:
Thanks for thinking to add a test!

It would be nice for this to get its own isolated test case though so that it doesn't get lost in the future.

Would you mind creating a `ProtocolParserGeneratorTest.kt` file in `src/test/kotlin/... same path as ProtocolParserGenerator .../` that tests this in isolation? Here's an example of how to do a client integration test that runs the full client code generator against a test model and verifies it compiles:
https://github.com/smithy-lang/smithy-rs/blob/c4a14d7c28fa26fb7f89f9634933965bab561cac/codegen-client/src/test/kotlin/software/amazon/smithy/rust/codegen/client/smithy/protocols/AwsQueryTest.kt#L12

Reply:
Sure can, would you like me to also revert the changes made in the current file or should I leave that in?

Reply:
Yeah, please revert.

Reply:
Done and done 😄